Stop considering Warnings experimental and add more cross-links. (#17738)
coq/coq: doc/sphinx/proof-engine/vernacular-commands.rst

.. opt:: Warnings "{+, {? {| - | + } } @ident }"

>    This :term:`option` configures the display of warnings. It
>    expects, between quotes, a comma-separated list of warning names or
>    categories. Adding - in front of a warning or category disables it, adding +

Reviewer: ```suggestion
   Configures the display of warnings. The :n:`@ident`s are warning names or
   categories. Adding - in front of a warning or category disables it, adding +
```

   makes it an error. It is possible to use the special categories `all` and

>    `default`, the latter containing the warnings enabled by default. The flags are

Author: Now that #17585 has been merged, this becomes reasonable to provide such a list.

>    interpreted from left to right, so in case of an overlap, the flags on the

Reviewer: Also, the doc should mention that, IIUC, in the warning below `unsupported-attributes` is the warning name and `parsing` is the warning category.  I'm not aware that this detail is covered anywhere.

`This command does not support this attribute: warnings. [unsupported-attributes,parsing]`
